### DISCUSSION TITLE
How to view the hyperlink description of a site column of type 'Link'

### DISCUSSION THREAD

**Discussion by woscheer** [microsoft-search/pnp-modern-search#3335] [Q&A]
Hello,
is it possible to view the description of a hyperlink column in the search result webpart?
Normally the hyperlink column is constructed with URLcommaDESCRIPTION and it's easy to pull out the description part.
When this column is mapped to a RefinableString managed property only the Url-part is populated there.

Debug View:
![pnpsearch01](https://github.com/microsoft-search/pnp-modern-search/assets/55751497/3fb19573-868b-409b-9e98-6b224d6440a5)

and there is no <SiteColumnName>OWSURLH crawled property generated like in SharePoint onPrem, which can be additionally added to the managed property.

So is it somehow possible to display the descriptive part of a hyperlink column?

Thanks

**Comment by woscheer** [comment]
It seemed that smth was going wrong while the managed property was created, another Hyperlink Site Column worked as described

**Reply by kasperbolarsen** [reply]
Nice to hear, this issue had me scratching my head as I know it used to work fine.
Thanks for updating 🙂
